flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/nodes/CygnusApplication.java

```diff
@@ -33,6 +33,7 @@
 import java.lang.reflect.Field;
```

Comment:
Fixed in commit https://github.com/telefonicaid/fiware-cygnus/commit/d8963381c6093f0fccbad6562affc886e3faf907